protocol/message.go

```diff
@@ -106,6 +181,14 @@ func NewKeyLookupProof(ap *m.AuthenticationPath, str *m.SignedTreeRoot,
 	}, e
 }
 
+// NewKeyLookupInEpochProof creates the response message a CONIKS server
+// sends to a client upon a KeyLookupInEpochRequest.
+// The contained DirectoryProofs is either a single proof of inclusion
+// (e = Success), or a proof of absence if the requested name does not
+// exist in the directory (e = ErrorNameNotFound).
+// In either case, the server returns a list of STRs for the epoch range [ep,
+// directory.LatestEpoch().Epoch], where ep is the past epoch for which
+// the client has requested the user's key.
```

Comment:
Oh sorry, reading my comment, I was confused myself... Sorry!

What I meant is that *descriptions* of behaviour of the server should be moved where this behaviour happens; for instance, comments like the above: "In either case, the server returns a list of STRs for the epoch range" should appear above methods where the STRs are actually generated (the comment of `KeyLookupInEpoch` should talk about the STRs and not the necessarily this constructor where they are just passed as an argument and aren't generated/returned).

Does this make more sense?